### PR TITLE
Update Cheesechain Routes

### PR DESCRIPTION
**Pull request by bivsk**
### Description

Update CheeseChain USDC route

### CODE REVIEW

**Review by jmrossy — CHANGES_REQUESTED** [review]
A few problems here:
1) if the route is cheesechain <-> arbitrum, it should not be changing this existing cheesechain <-> ethereum file
2) The PR title is inaccurate  
3) You're missing a changeset (run `yarn changeset`)

**Comment by bivsk** [comment]
Regarding 1, we do not want to support the old cheesechain <-> ethereum route any longer. Can I just rename this current file?

**Comment by rshtirmer** [comment]
> A few problems here:
> 
> 1. if the route is cheesechain <-> arbitrum, it should not be changing this existing cheesechain <-> ethereum file
> 2. The PR title is inaccurate
> 3. You're missing a changeset (run `yarn changeset`)

Hey J,

I published the original route that @bivsk is overwriting - that route being overwritten is deprecated, and does not point to what is now the canonical USDC on cheesechain (which is created by bridging USDC from arb -> cheesechain via this new route)  - @bivsk will add a changeset, but otherwise I think this should be ok? Please let me know if theres anything else we can do to support. 

Best,
Ryan Shtirmer

**Comment by bivsk** [comment]
@jmrossy I've made a few changes to hopefully fix the above issues, while also sticking to our plan to replace rather than add a route.  Hopefully this works, but let me know if not.  Thanks a lot, and have a good rest of your weekend

File: .changeset/chilled-yaks-nail.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```
Comment (jmrossy) [inline .changeset/chilled-yaks-nail.md:2]:
```suggestion
'@hyperlane-xyz/registry': major
```

Comment (jmrossy) [reply .changeset/chilled-yaks-nail.md:2]: Whenever something is removed, that's a breaking change and the version increment will need to be `major`

File: deployments/warp_routes/USDC/cheesechain-arbitrum-config.yaml
Comment (jmrossy) [inline deployments/warp_routes/USDC/cheesechain-arbitrum-config.yaml:1]: Ditto above for filename

File: deployments/warp_routes/USDC/cheesechain-arbitrum-addresses.yaml
Comment (jmrossy) [inline deployments/warp_routes/USDC/cheesechain-arbitrum-addresses.yaml:1]: The filename is now incorrect. The chains should be alphabetically ordered (arbitrum-cheesechain).

**Review by jmrossy — COMMENTED** [review]
Almost there!

**Comment by bivsk** [comment]
OK pretty sure it's all good to go now 😅

Thanks again!

**Comment by jmrossy** [comment]
@bivsk Still one more problem. run `yarn lint:fix`

Screenshot 2024-10-20 at 12 50 18 PM

**Review by jmrossy — APPROVED** [review]

**Pull request merged**